### PR TITLE
utils: Make get_last_address_from_wallet() backward compatible

### PR DESCRIPTION
**Pull request by evgeniiz321**
Relates to #66, tests at neofs-testcases configure wallets with different amount of accounts, when there is more than 1 account tests fail.

### CODE REVIEW

File: src/neofs_testlib/utils/wallet.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import json
 import logging
 
-from neo3.wallet import wallet as neo3_wallet
 from neo3.wallet import account as neo3_account
+from neo3.wallet import wallet as neo3_wallet
```
Comment (roman-khimov) [inline src/neofs_testlib/utils/wallet.py:5]: Useless hunk, just don't change it please.

File: tests/test_wallet.py [hunk 2]
```diff
@@ -3,9 +3,10 @@
 from unittest import TestCase
 from uuid import uuid4
 
+from neo3.wallet import account as neo3_account
 from neo3.wallet.wallet import Wallet
 
-from neofs_testlib.utils.wallet import init_wallet, get_last_address_from_wallet
+from neofs_testlib.utils.wallet import get_last_address_from_wallet, init_wallet
```
Comment (roman-khimov) [inline tests/test_wallet.py:9]: Doesn't seem to be a useful change as well.

**Review by roman-khimov — CHANGES_REQUESTED** [review]
Looks legit otherwise.

File: src/neofs_testlib/utils/wallet.py (hunk 1 above)
Comment (evgeniiz321) [reply src/neofs_testlib/utils/wallet.py:5]: this is from a black formatter, that configured like said in contribute.md. Remove it?

Comment (roman-khimov) [reply src/neofs_testlib/utils/wallet.py:5]: Either drop, or make a separate formatting commit with no functional changes.

Comment (evgeniiz321) [reply src/neofs_testlib/utils/wallet.py:5]: Made a separate commit

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**